Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h

```diff
@@ -405,6 +412,12 @@ namespace AzToolsFramework
     public:
         typedef WidgetType widget_t;
 
+        // resets widget attributes and properties for reuse
+        virtual void ResetGUIToDefaults(WidgetType* widget)
+        {
+            (void)widget;
+        }
```

Comment:
Could also do 
```suggestion
        virtual void ResetGUIToDefaults([[maybe_used]] WidgetType* widget)
        {
            (void)widget;
        }
```

Reply:
In this case, I tried to keep the file consistent, but I can change it to our newer convention.